airbyte-integrations/connectors/destination-meilisearch/README.md

```diff
@@ -0,0 +1,33 @@
+# MeiliSearch Destination
+
+This is the repository for the MeiliSearch destination connector, written in Java.
+For information about how to use this connector within Airbyte, see [the documentation](https://docs.airbyte.io/integrations/destomatopms/meilisearch).
```

Comment:
is this url correct?

Reply:
woof. i don't even know what happened there.

Reply:
fixed in separate pr.